# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/nsDOMWindowUtils.cpp
# Commit: a0daec0e7301
# Full Hash: a0daec0e73011a6b972d2bc87c1cf620d569af80
# Author: Kris Taeleman <ktaeleman@mozilla.com>
# Date: 2019-09-27 04:28:18
# Regressor Bug: 1582565
# File Overlap Count: 6
# Description:
#   Bug 1582565 - Log transactions into readable format. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D46657
# ==============================================================================

diff -r 3bd01ec4a9e9 -r a0daec0e7301 dom/base/nsDOMWindowUtils.cpp
--- a/dom/base/nsDOMWindowUtils.cpp	Wed Sep 25 20:46:24 2019 +0000
+++ b/dom/base/nsDOMWindowUtils.cpp	Thu Sep 26 16:43:05 2019 +0000
@@ -4069,6 +4069,12 @@
   return NS_OK;
 }
 
+NS_IMETHODIMP
+nsDOMWindowUtils::SetTransactionLogging(bool aValue) {
+  Preferences::SetBool("gfx.webrender.debug.log-transactions", aValue);
+  return NS_OK;
+}
+
 void nsDOMWindowUtils::ReportErrorMessageForWindow(
     const nsAString& aErrorMessage, const char* aClassification,
     bool aFromChrome) {